src/components/pages/team/hero/team.jsx

```diff
@@ -37,27 +52,78 @@ const Team = ({ info }) => {
     });
   }, 500);
 
+  const leaveTeam = async () => {
+    if (!currentUser) return;
+
+    if (confirm('Are you sure want to leave the team?')) {
+      try {
+        const response = await fetch('/api/leave-team', {
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+          },
+          method: 'POST',
+          body: JSON.stringify({
+            id: currentUser.id,
+          }),
+        }).then((res) => res.json());
+
+        if (response.success === true) {
+          toast.success(response.message);
+          router.push('/joinsquad');
```

Comment:
I think you wante "/myteam" ?

Reply:
@nevo-david since we are already on the same team, the page won't be refresh.
So I thought to redirect to this page instead.

Reply:
use window.reload() then :)